explorer/app/components/Project/common/projectTransformer.ts

```diff
@@ -297,6 +300,34 @@ export function getProjectTitle(
   return project.projectTitle;
 }
 
+/**
```

Comment:
Let's rollback this whole file `projectTransformer`. So remove `getProjectIconList` method and revert the changes in the `getProjectDescription` function.